### DISCUSSION TITLE
YOURLS Blog SSL Cert Expired

### DISCUSSION THREAD

**Discussion by hughamacmullaniv** [YOURLS/YOURLS#3441] [General]
The [YOURLS Blog](https://blog.yourls.org/) SSL cert expired a couple of days ago:
`Expires On	Thursday, October 13, 2022 at 5:53:32 PM
`
FYI! I predict Let's Encrypt cron fails because `0 0 ...`

**Comment by ozh** [comment]
Oh, thanks for the heads up. I'll have a look !
